resources/css/app.css

```diff
@@ -1,3 +1,5 @@
+@import 'charts.css';
```

Comment:
where is this file?

Reply:
a new file in package.json named that way.